test/handler.test.js

```diff
@@ -199,9 +200,31 @@ describe('strong-error-handler', function() {
       requestJson().end(function(err, res) {
         if (err) return done(err);
 
-        var expectedData = {statusCode: 500, stack: error.stack};
-        for (var key in error) expectedData[key] = error[key];
+        var expectedData = {
+          statusCode: 500,
+          message: 'a test error message',
+          name: 'ErrorWithProps',
+          details: 'some details',
+          extra: 'sensitive data',
+          stack: error.stack,
+        };
+        expect(res.body).to.have.property('error');
+        expect(res.body.error).to.eql(expectedData);
+        done();
+      });
+    });
 
+    it('contains name property when debug=true', function(done) {
```

Comment:
because when the [`testHelper`](https://github.com/strongloop/strong-error-handler/blob/42613b735389d71412c69cae60a9c51be5c59741/test/handler.test.js#L423-L432) does it, it doesn't make it `enumerable=false` so I was thinking it could be helpful to have it there.